compatibility/bazel_tools/testing.bzl

```diff
@@ -373,6 +373,18 @@ def create_daml_app_test(
         **kwargs
     )
 
+# FIXME
+#
+# From version 1.11.0-snapshot.20210217.6338.1 of the SDK, the Ledger API
+# Test Tool uses Daml-LF 1.11, which requires platform versions starting from
+# 1.10.0.
+#
+# This predicate is used to filter Ledger API Test Tool tests in the
+# sdk_platform_test macro as a temporary measure to prevent spurious errors on CI.
+# The proper fix is to use the appropriate version of Daml-LF for every SDK/platform pair.
+def ledger_api_test_tool_compatible(sdk_version, platform_version):
+    return in_range(sdk_version, {"end": "1.11.0-snapshot.20210217.6338.1"}) or (in_range(sdk_version, {"start": "1.11.0-snapshot.20210217.6338.1"}) and in_range(platform_version, {"start": "1.10.0"}))
```

Comment:
AFAIK `in_range` is inclusive on both ends. So, this would include a pairing of `sdk_version == 1.11.0-snapshot.20210217.6338.1` with say `platform_version == 1.9.0`. But, as I understand the comment, that would be incompatible.

Reply:
`1.11.0-snapshot.20210217.6338.1` does not exist, but sorts before the first release that uses 1.11 and after the last release that uses 1.8. So this is in effect an excluded bound on `1.11.0-snapshot.20210217.6338.0.ba6ba901`.

Reply:
Ugh... thanks, I'll take care of this.

Reply:
fwiw I think the current inclusive on both behavior makes no sense (and I say that as the person having implemented it). I think we should be exclusive on the start end inclusive on the end.  That works for the cases where we make a change and want it to apply to everything after the last snapshot.

Reply:
If you do change the inclusivity of bounds, perhaps change the name of the fields too? `start-after` and `end-at`, perhaps?